test/functional/apps/console/index.ts

```diff
@@ -17,13 +17,16 @@
  * under the License.
  */
 
-export default function ({ getService, loadTestFile }) {
+import { FtrProviderContext } from '../../../types';
+
+// tslint:disable-next-line no-default-export
```

Comment:
Could we perhaps disable this check completely for the function tests, since it's just the way the functional test framework has been build and even new test files will have to use that, so I think we should rather disable it in tslint for the `test` folder than to force every file to have that tslint disable in it, since I think an inline tslint-disable should be for exceptional cases, and ideally should go with a comment why it needs to be disabled there. This is not an exceptional case, but the expected behavior.

Reply:
Not really, tslint doesn't support overrides for patterns or anything like eslint does. When https://github.com/typescript-eslint/typescript-eslint is ready and we move to it we will be able to, but until then this is our best option without dropping tslint.yaml files all over the place with I think is even worse.